source/06_monitoring_features/network_reachability.rst

```diff
@@ -0,0 +1,156 @@
+to do this. Here's what the (abbreviated) host definitions with parent/child relationships would look like for this example:
+
+  
+::
+
+  ; This host has no parent - it is the topmost host
+  define host{
+    host_name       Alignak
+  }
+  
+  define host{
+    host_name       Switch1
+    parents         Alignak
+  }
+  
+  define host{
+    host_name       Web
+    parents         Switch1
+  }
+  
+  define host{
+    host_name       FTP
+    parents         Switch1
+  }
+  
+  define host{
+    host_name       Router1
+    parents         Switch1
+  }
+  
+  define host{
+    host_name       Switch2
+    parents         Router1
+  }
+  
+  define host{
+    host_name       Wkstn1
+    parents         Switch2
+  }
+  
+  define host{
+    host_name       HPLJ2605
+    parents         Switch2
+  }
+  
+  define host{
+    host_name       Router2
+    parents         Router1
+  }
+  
+  define host{
+    host_name       somewebsite.com
+    parents         Router2
+  }
+
+
+Reachability logic in action
+============================
+
+Now that you have configured Alignak with the proper parent/child relationships for your hosts, let's see
+what happen when problems arise. Assume that two hosts - Web and Router1 - go offline...
+
+
+.. image:: /_static/images///official/images/reachability3.png
+   :scale: 90 %
+
+
+When hosts change state (i.e. from UP to DOWN), the host reachability logic in Alignak kicks in.
+The reachability logic will initiate parallel checks of the parents and children of whatever hosts change state.
+This allows Alignak to quickly determine the current status of your network infrastructure when changes occur.
+
+
+.. image:: /_static/images///official/images/reachability4.png
+   :scale: 90 %
+
+
+In this example, Alignak will determine that Web and Router1 are both in DOWN states because the "path" to those hosts is not being blocked.
+
+Alignak will determine that all the hosts "beneath" Router1 are all in an UNREACHABLE state because Alignak can't
+reach them. Router1 is DOWN and is blocking the path to those other hosts.
+Those hosts might be running fine, or they might be offline - Alignak doesn't know because it can't reach them.
+Hence Alignak considers them to be UNREACHABLE instead of DOWN.
+
+
+UNREACHABLE states and notifications
+====================================
+
+By default, Alignak will notify contacts about both DOWN and UNREACHABLE host states. As an admin/tech,
+you might not want to get notifications about hosts that are UNREACHABLE. You know your network structure,
+and if Alignak notifies you that your router/firewall is down, you know that everything behind it is unreachable.
+
+If you want to spare yourself from a flood of UNREACHABLE notifications during network outages, you can
+exclude the unreachable (u) option from the `notification_options` directive in your host definitions and/or the
```

Comment:
We should include more information about this in the configuration part of the doc ...